bridge/matrix/matrix.go

```diff
@@ -257,47 +259,53 @@ func (b *Bmatrix) handleDownloadFile(rmsg *config.Message, content map[string]in
 	return nil
 }
 
-// handleUploadFile handles native upload of files
-func (b *Bmatrix) handleUploadFile(msg *config.Message, channel string) (string, error) {
+// handleUploadFiles handles native upload of files
```

Comment:
Nit: we should make all these comments into real sentences with a period at the end.